nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractElasticsearchHttpProcessor.java

```diff
@@ -316,9 +341,12 @@ protected void buildBulkCommand(StringBuilder sb, String index, String docType,
         if (indexOp.equalsIgnoreCase("index")) {
             sb.append("{\"index\": { \"_index\": \"");
             sb.append(StringEscapeUtils.escapeJson(index));
-            sb.append("\", \"_type\": \"");
-            sb.append(StringEscapeUtils.escapeJson(docType));
             sb.append("\"");
+            if (!(StringUtils.isEmpty(docType) | docType == null)){
```

Comment:
Should probably use isBlank rather than isEmpty her and no need for the additional null (also use ||, not |)